src/vdfs.c

```diff
@@ -29,6 +29,7 @@
 #include "mem.h"
 #include "model.h"
 #include "tube.h"
+#include "compat_wrappers.h"
```

Comment:
Ah, I didn't spot that. This line can be removed then. I'll do that tomorrow and re-submit.